src/toil/test/src/jobServiceTest.py

```diff
@@ -109,7 +109,7 @@ def runToil(self, rootJob, retryCount=1, badWorker=0.5, badWorkedFailInterval=0.
         options.logLevel = "INFO"
         
         options.retryCount = retryCount
-        #options.clean = "never"
+        options.clean = "never"
```

Comment:
Because when it is commented the testService test fails with 
```
JobStoreCreationException: The job store '/var/folders/pk/r00rdz2d5_lcjkffz5y5xmnc0000gn/T/toil-test-toil.test.src.jobServiceTest.JobServiceTest.testService-jobstore-7iBTK_' does not exist, so there is nothing to restart.
```
One of the toil runs in this test requires that a preceding run not be cleaned up